rcljava/src/main/java/org/ros2/rcljava/action/CancelCallback.java

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2020 ros2-java contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ros2.rcljava.action;
+
+import org.ros2.rcljava.interfaces.ActionDefinition;
+
+public interface CancelCallback<T extends ActionDefinition> {
+  enum CancelResponse {
```

Comment:
should it be public?

Reply:
IIUC, all members of a Java interface are public (cannot be otherwise). According to the [language spec](https://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.4), adding a `public` modifier here is okay, but discouraged.